content/client-lib-development-guide/features.textile

```diff
@@ -1701,6 +1736,26 @@ class ChannelOptions:
   params?: Dict<String, String> // TB2c
   modes?: [ChannelMode] // TB2d
 
+class ChannelDetails:
+  channelId: String //CHD2a
```

Comment:
We should add a space in between the comment symbol and the comment (ie `//CHD2a` -> `// CHD2a`) for consistency with the rest of the comments in this section.

Reply:
Done

Reply:
There are still some more places where this needs to be changed

Reply:
Is there a way to auto format this document?